styles.css

```diff
@@ -0,0 +1,197 @@
+nav {
+  position: relative;
+  z-index: 99 !important;
+  background-color: #fff;
+}
+
+.nav1 {
+  width: 97%;
+  margin: auto;
+  background-color: #f72210;
+  color: #fff;
+}
+
+.nav1 p {
+  margin: auto 30px;
+}
+
+.title {
+  color: #fff;
+  font-size: 90px;
+  font-weight: bold;
+}
+
+.title:hover {
+  text-decoration: none;
+  color: #fff;
+}
+
+.nav2,
+.nav3 {
+  width: 97%;
+  margin: 10px auto;
```

Comment:
- [ ] Use bootstrap for this, [learn more](https://getbootstrap.com/docs/4.0/utilities/spacing/)